### PR TITLE
config: allow to run bump year manually

### PR DESCRIPTION
**Pull request by romani**
allow to run bump year manually

### CODE REVIEW

File: .github/workflows/bump_license_year.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@
 #############################################################################
 name: "Bump license year"
 on:
+  workflow_dispatch: null
```
Comment (nrmancuso) [inline .github/workflows/bump_license_year.yml:9]: Please specify branch here, like:
```
workflow_dispatch:
  branches: [ master ]

```
If cron only runs on master, so should dispatched job.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: .github/workflows/bump_license_year.yml (hunk 1 above)
Comment (romani) [reply .github/workflows/bump_license_year.yml]: done

Comment (nrmancuso) [reply .github/workflows/bump_license_year.yml:9]: Branches option for workflow_dispatch sees to deprecated, actionlint failed on this. I restored `null`.

**Review by nrmancuso — APPROVED** [review]

**Comment by nrmancuso** [comment]
Usual failure in travis: `No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`

**Pull request merged**